page-moh-about.php

```diff
@@ -1,30 +1,33 @@
 <?php
-
-/* Template Name: MOH About */
-
+/**
+ * Template Name: MOH Full Width
```

Comment:
This `MOH Full Width` name is currently shared between two templates (page-full-no.php and page-moh-about.php), which may make one not possible to select.

Reply:
thanks